### DISCUSSION TITLE
How to open telescope with normal mode

### DISCUSSION THREAD

**Discussion by thekaganugur** [stevearc/dressing.nvim#53] [Q&A]
I want to open telescope with `initial_mode=normal` since most of the time I won't do a search but rather select with `j` and `k`.

I couldn't find a way to do it. Any help apricated.

**Comment by thekaganugur** [comment]
One more thing I'm having a problem transitioning from `coc`, is there a way to sort actions? In `coc` auto-imports were always on most top. This is probably not related to `dressing.nvim` but help would be really good.

**Comment by stevearc** [comment]
The `telescope` field of the config accepts all normal telescope options, so you can simply do
```lua
require('dressing').setup({
  select = {
    telescope = {
      initial_mode = "normal"
    }
  }
})
```

As for sorting code actions, you could do something like:
```lua
require('dressing').setup({
  select = {
    get_config = function(opts, items)
      if opts.kind == "codeaction" then
        -- sort items
      end
    end,
  }
})
```
Note that for the sorting it requires a commit that I just pushed, so you'll need to git pull.
